exchanges/exchange.go

```diff
@@ -1500,7 +1500,7 @@ func (b *Base) IsPerpetualFutureCurrency(asset.Item, currency.Pair) (bool, error
 
 // GetKlineRequest returns a helper for the fetching of candle/kline data for
 // a single request within a pre-determined time window.
-func (b *Base) GetKlineRequest(pair currency.Pair, a asset.Item, interval kline.Interval, start, end time.Time) (*kline.Request, error) {
+func (b *Base) GetKlineRequest(pair currency.Pair, a asset.Item, interval kline.Interval, start, end time.Time, setTimeWindow bool) (*kline.Request, error) {
```

Comment:
I think `fixedAPICandleLength bool` is more clear about what you're implementing

Reply:
Yeah that is a good suggestion 🔥